modules/hrm/assets/js/requests.js

```diff
@@ -207,10 +207,10 @@
 
                 tableHeaders: function() {
                     let headers = [];
-                    
```

Comment:
In here, this extra space should be removed if possible. and same as some bottom codes also. It would be removed if someone use Extra space remover plugin. For any confusion, please see this screenshot - https://prntscr.com/1lgovty

Reply:
Where did you find those spaces?
I didn't find any neither in [here](https://prnt.sc/1lofyqv) not in my Editor.